site/src/docs/about/resources.mdx

```diff
@@ -35,3 +35,7 @@ import InternalLink from '../../components/InternalLink';
 - <ExternalLink href="https://github.com/City-of-Helsinki/helsinki-design-system/tree/master/packages/react">HDS React</ExternalLink>
 - <ExternalLink href="https://github.com/City-of-Helsinki/helsinki-design-system/tree/master/packages/design-tokens">HDS Design tokens</ExternalLink>
 - <ExternalLink href="https://github.com/City-of-Helsinki/helsinki-design-system/tree/master/site">HDS Documentation site</ExternalLink>
+
+## HDS storybook examples
+- <InternalLink href="https://hds.hel.fi/storybook/react">React Storybook</InternalLink>
```

Comment:
As they are internal links, the `href` should be a relative path like in other internallinks:  `<InternalLink href="/storybook/react"`

Reply:
Good point, I'll fix